docs/development/dockerbuildmaint.rst

```diff
@@ -15,12 +24,16 @@ to do so, you'll need to make sure your local docker client has credentials to p
   obtained from the previous step.
 * Proceed with update instructions
 
+Updating the build container
+============================
+We know the build container needs to be updated when **test_ensure_no_updates_avail** fails during
+``make build-debs`` in the ``securedrop`` Github reprository root directory. This test fails if any
+of the dependencies required to build the debian packages have security updates. If you have access
+rights to push to quay.io, then you can build and push a new container to the Quay repository by
+following the steps below.
 
-Performing container updates
-----------------------------
-If one of the dependencies requires security updates, the build may fail at
-**test_ensure_no_updates_avail** . If you have access rights to push to quay.io,
-here is the process to build and push a new container:
+.. note:: The reason we don't update the container at runtime is that we use the container image as
+          a way of recording our build environment.
```

Comment:
Could equivalent security properties be achieved by including the exact version information in the build logs automatically, as suggested in https://github.com/freedomofpress/securedrop/issues/4533#issuecomment-705717195 ? If so, it does not seem a compelling argument to include in favor of the current process.

Reply:
The docs presented here are definitely accurate in terms of the process we actually adhere to. I remain interested in getting the SD core debs fully reproducible, so we can stop fussing with the build container so much. After Focal, that seems a reasonable goal again, consistent with discussion in https://github.com/freedomofpress/securedrop/issues/4533. 

All that to say, the "note" as written here LGTM!